src/js/entities-service/entities/form-responses.js

```diff
@@ -42,6 +42,9 @@ const _Model = BaseModel.extend({
   getResponse() {
     return get(this.get('response'), 'data', {});
   },
+  getFormData() {
```

Comment:
Should this be `getResponseData`? Although that's confusing with `getResponse`...

Reply:
it's all a bit confusing in terms, but the object that get's preloaded is `formData` so this is kind of the response's `formData`. but `response.data` is also a thing.. so.. it's not entirely obvious without renaming a bunch of stuff.

Reply:
Yeah, that's true. There isn't a more obvious name, so it works.